scalalib/src/ScalaModule.scala

```diff
@@ -57,6 +58,8 @@ trait ScalaModule extends JavaModule { outer =>
    */
   def scalaVersion: T[String]
 
+  def scalaOutputVersion: T[String] = scalaVersion
```

Comment:
Here we need some scaladoc explaining its purpose and also the distinction to `scalaVersion`.